src/views/LoanDetails/LoanDetails.test.js

```diff
@@ -371,13 +387,13 @@ describe('LoanDetails', () => {
       mockAccounts.mockImplementation((item1, item2) => {
         return item2 === 'open';
       });
-      const updatedPropsData = { ...propsData, loanAccountActions: [{}, {}] };
+      const updatedPropsData = { ...claimedReturnedPropsData, loanAccountActions: [{}, {}] };
       renderLoanProxyDetails(updatedPropsData);
 
       userEvent.click(screen.queryAllByRole('button')[5]);
       expect(screen.getAllByText('ui-users.loans.markAsMissing')).toBeTruthy();
     });
-    it.skip('Fee Fine Else Condition More than 1 loanAccountActions with new loan', () => {
+    it('Fee Fine Else Condition More than 1 loanAccountActions with new loan', () => {
```

Comment:
Could you please improve the test title to better elucidate the aim of the unit test?